src/main/java/duke/commands/AddCommand.java

```diff
@@ -0,0 +1,94 @@
+package duke.commands;
+import duke.utils.*;
+import duke.tasks.*;
+
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+
+public class AddCommand extends Command {
+    private String type;
+    private String fullCommand;
+
+    public AddCommand(String type, String fullCommand) {
+        this.type = type;
+        this.fullCommand = fullCommand;
+    }
+
+    /**
+     * Creates a new task and adds it to the list.
+     *
+     * @param storage
+     * @param tasks
+     */
+    public void execute(Storage storage, ArrayList<Task> tasks) {
+        String[] words = fullCommand.split(" ");
+
+        try {
+            int startIndex;
+            String description;
+            Task newTask = new Task("");
+            switch (type) {
+                case "todo":
```

Comment:
Any reason why u didn't add a time option for this like the other event types

Reply:
todo tasks don't have dates attached to them.